### PR TITLE
Update documentation for MemCacheStore::build_mem_cache

### PR DESCRIPTION
**Pull request by davydovanton**

### CODE REVIEW

File: activesupport/lib/active_support/cache/mem_cache_store.rb [hunk 1]
```diff
@@ -26,6 +26,13 @@ module Cache
     class MemCacheStore < Store
       ESCAPE_KEY_CHARS = /[\x00-\x20%\x7F-\xFF]/n
 
+      # Creates a new Dalli::Client instance with specified addresses and options.
+      # By default address is equal localhost:11211.
+      #
+      #   ActiveSupport::Cache::MemCacheStore.build_mem_cache()
```
Comment (rafaelfranca) [inline activesupport/lib/active_support/cache/mem_cache_store.rb]: omit the parentheses

File: activesupport/lib/active_support/cache/mem_cache_store.rb [hunk 2]
```diff
@@ -26,6 +26,13 @@ module Cache
...
+      #     # => #<Dalli::Client:0x007f98a47d2028 @servers=["localhost:11211"], @options={}, @ring=nil>
+      #   ActiveSupport::Cache::MemCacheStore.build_mem_cache('localhost:10290')
+      #     # => #<Dalli::Client:0x007f98a47b3a60 @servers=["localhost:10290"], @options={}, @ring=nil>
       def self.build_mem_cache(*addresses)
```
Comment (rafaelfranca) [inline activesupport/lib/active_support/cache/mem_cache_store.rb]: I think this method should not be public. Could you add a `# :nodoc:` here?

**Comment by davydovanton** [comment]
@rafaelfranca I added `:nodoc:` for `::build_mem_cache` and also add information about `Dalli::Client` to `MemCacheStore#initialize`. What do you think, it's normal for you?

**Comment by rafaelfranca** [comment]
Sorry, I was not clear. You can keep the `build_mem_cache` documentation you wrote and add the `# :nodoc:`. Even private API methods can be documented.

I think we can also remove the `Dalli::Client` change since it is just an implementation detail.

**Comment by davydovanton** [comment]
@rafaelfranca oh, sorry, I understood you :sweat:
Fixed all

**Comment by rafaelfranca** [comment]
thank you so much

**Comment by davydovanton** [comment]
thank you for review my PR :+1:

**Pull request merged**